packages/foundry-react-ui/src/components/InteractionFeedback/InteractionFeedback.tsx

```diff
@@ -36,7 +36,8 @@ export type InteractionFeedbackProps = {
 };
 
 const defaultInterpolationFunctions = {
-  r: (r: any) => r.to((val: string) => `${Math.abs(parseFloat(val))}`),
+  r: (r: any) => r.to((val: string) => `${Math.abs(parseFloat(val).toFixed(2))}`),
```

Comment:
I'm seeing a type error on my end, `toFixed` returns a string which isn't accepted by `Math.abs`.

Reply:
Nice catch. I'm a rubyist so it would have either worked by some sort of ✨ Rails Magic ✨, resulted in a rounding error that only happens 1-out-of-10000 calculations or would have completely taken down the entire application. ;)

Reply:
Haha yeah and JavaScript will absolutely figure this out, but TypeScript just keeping us honest and not relying on JS magic too much.